project_start.py

```diff
@@ -99,6 +100,50 @@ def read_goals(tsv):
     return dict(goals), fulfillments
 
 
+def sanitize(unsanitized):
+    """Sanitize a dict or list containing strings.
+
+    Parameters
+    ----------
+    unsanitized : iterator
+        Dictionary or list to sanitize
+
+    Returns
+    -------
+    iterator
+        Copy of input with sanitized strings
+    """
+    sanitized = None
+    if isinstance(unsanitized, dict):
+        sanitized = {}
+        for k, v in unsanitized.items():
+            sanitized[sanitize_string(k)] = sanitize_string(v)
+    elif isinstance(unsanitized, list):
+        sanitized = []
```

Comment:
Not a fan of list comprehension?
```suggestion
        sanitized = [sanitize_string(i) for i in unsanitized]
```